Project/src/main/java/com/github/lgooddatepicker/components/DatePickerSettings.java

```diff
@@ -1307,14 +1319,13 @@ public void setBorderPropertiesList(ArrayList<CalendarBorderProperties> borderPr
   }
 
   /**
-   * setCalendarPanelFactory, This sets the factory used to construct instances of CalendarPanel. If
-   * it is null the standard CalendarPanel is instantiated.
-   *
+   * setCalendarPanelFactory, This sets the factory used to construct instances of CalendarPanel. If it is null
+   * the standard CalendarPanel is instantiated.
    * @param calendarPanelFactory
    */
   public void setCalendarPanelFactory(Function<DatePicker, CalendarPanel> calendarPanelFactory) {
     this.calendarPanelFactory = calendarPanelFactory;
-  }
+}
```

Comment:
The alignment of the closing method's closing brace does not line up with `public void ...`  and it did previously